gui/control_main.py

```diff
@@ -2949,12 +2952,18 @@ def fillPolyRaster(
         self.currentRasterCellList = currentRasterGroup.childItems()
         cellResults = rasterResult["result_obj"]["rasterCellResults"]["resultObj"]
         numLines = len(cellResults)
-        cellResults_array = [{} for i in range(numLines)]
+        # cellResults_array = [{} for i in range(numLines)]
         my_array = np.zeros(numLines)
         spotLineCounter = 0
         cellIndex = 0
         rowStartIndex = 0
         rasterEvalOption = str(self.rasterEvalComboBox.currentText())
+        if rasterEvalOption == "Spot Count":
```

Comment:
if you expect changing of what the GUI shows as fields to evaluate and/or what the results have as keys, you may want to put the options and keys into a dictionary, and use this to populate the ComboBox

Reply:
I could do this in a different PR as this would need changes in multiple places not related to this one